lms/static/styles/_variables.scss

```diff
@@ -6,6 +6,11 @@ $grey-2: #ececec;
 $grey-3: #dbdbdb;
 $grey-4: #a6a6a6;
 $grey-5: #7a7a7a;
+// Interim color variable for migration purposes, as the step between `$grey-5`
+// and `$grey-6` is large. Represents `base-mid` in proposed future palette,
+// minus blue tint.
+// This is the lightest grey admissible on $grey-2, $grey-3
+$grey-mid: #595959;
```

Comment:
Pulled this terrible color in form the sidebar, It really need it  unfortunately  since it is the button color over there and its just too light if we use $grey-5.

Reply:
Some alternative:

```suggestion
$grey-5\.5: #595959;
```

```suggestion
$grey-5-5: #595959;
```

```suggestion
$grey-5-tmp: #595959;
```

Reply:
Why is this an interim color? Why don't we fully adopt it?

If we would adopt this new grey value, it may be makes sense to convert all:
*  `grey-7` to `grey-8`
* `grey-6` to `grey-7`

Reply:
I think we do revisit this rule next week. It was dropped in to the client as  a shim many moons ago, but we did not expect it to be long lived. We were at that time in the process of developing a new color palette but that was put on hold so its long outlived it expected life span.

We should keep the name the same as the client (if we do end up using it) OR move to a new pallet entirely and then abandon it.